14th-team5-iOS/Bibbi/Sources/Presentation/Notification/ViewControllers/NotificationViewController.swift

```diff
@@ -0,0 +1,277 @@
+    
+    override init(
+        style: UITableViewCell.CellStyle,
+        reuseIdentifier: String?
+    ) {
+        super.init(
+            style: style,
+            reuseIdentifier: reuseIdentifier
+        )
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func setupUI() {
+        addSubviews(
+            profileView,
+            titleLabel,
+            timeLabel,
+            contentLabel
+        )
+    }
+    
+    override func setupAutoLayout() {
+        profileView.snp.makeConstraints {
+            $0.leading.equalToSuperview().inset(20)
+            $0.top.equalToSuperview().inset(14)
+        }
+        
+        titleLabel.snp.makeConstraints {
+            $0.top.equalToSuperview().inset(16)
+            $0.leading.equalTo(profileView.snp.trailing).offset(12)
+            $0.trailing.equalToSuperview().inset(20)
+        }
+        
+        timeLabel.snp.makeConstraints {
+            $0.top.equalTo(titleLabel)
+            $0.trailing.equalToSuperview().inset(20)
+        }
+        
+        contentLabel.snp.makeConstraints {
+            $0.top.equalTo(titleLabel.snp.bottom)
+            $0.leading.equalTo(titleLabel)
+            $0.trailing.equalTo(timeLabel)
+            $0.bottom.equalToSuperview().inset(16)
+        }
+    }
+    
+    override func setupAttributes() {
+        self.backgroundColor = .clear
+        
+        titleLabel.do {
+            $0.numberOfLines = 2
+        }
+        
+        contentLabel.do {
+            $0.numberOfLines = 0
+        }
+    }
+}
+
+extension NotificationCell {
+    private func bindInput(reactor: NotificationCellReactor) {
+        Observable.just(())
+            .map { Reactor.Action.setCell }
+            .bind(to: reactor.action)
+            .disposed(by: disposeBag)
+    }
+    
+    private func bindOutput(reactor: NotificationCellReactor) {
+        reactor.state.map { $0.profile }
+            .bind(to: profileView.rx.configure)
+            .disposed(by: disposeBag)
+        
+        reactor.state.map { $0.notification.title }
+            .bind(to: titleLabel.rx.text)
+            .disposed(by: disposeBag)
+        
+        reactor.state.map { $0.notification.content }
+            .bind(to: contentLabel.rx.text)
+            .disposed(by: disposeBag)
+        
+        reactor.state.map { $0.time }
+            .bind(to: timeLabel.rx.text)
+            .disposed(by: disposeBag)
+    }
+}
+
+extension NotificationCell {
+    internal func setReactor(reactor: NotificationCellReactor) {
+        self.reactor = reactor
+        
+        bindInput(reactor: reactor)
+        bindOutput(reactor: reactor)
+    }
+}
+
+final class NotificationViewController: BBNavigationViewController<NotificationReactor> {
+    private typealias RxDataSource = RxTableViewSectionedReloadDataSource<NotificationSectionModel>
+    
+    private let divider: UIView = UIView()
```

Comment:
```suggestion
    private let dividerView: UIView = UIView()
```
요기도 `dividerView`로 하면 좋을 것 같아용. :)

Reply:
넵